cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -1213,6 +1214,18 @@ func checkDataAndSavePairing(rootdir string) (bool, error) {
 	return subtle.ConstantTimeCompare(marker1, marker2) == 1, nil
 }
 
+// waitFile waits for the given file/device-node/directory to appear.
+var waitFile = func(path string, wait time.Duration, n int) error {
```

Comment:
also, it would be nice to have this helper in osutil eventually too, since I seem to remember similar code elsewhere in the codebase

Reply:
Yeah, I was thinking a bit about this, in order for it to move to osutils I think we will probably some sort of callback so that the UI can generate message about the waiting and control that. But agreed that this should probably also follwup material :) Thanks for the review!